src/test/java/io/github/ygojson/application/yugipedia/client/AbstractYugipediaClientTest.java

```diff
@@ -55,7 +61,7 @@ private Response<QueryResponse> doExecuteTestQueryRecentChanges(
 		return doExecuteTestQueryRecentChanges(null, null, grccontinue);
 	}
 
-	private Response<QueryResponse> doExecuteTestQueryRecentChanges(
+	public Response<QueryResponse> doExecuteTestQueryRecentChanges(
```

Comment:
Should stay private:
```suggestion
	private Response<QueryResponse> doExecuteTestQueryRecentChanges(
```